zebra/if_netlink.c

```diff
@@ -884,6 +885,8 @@ static int netlink_interface(struct nlmsghdr *h, ns_id_t ns_id, int startup)
 	ns_id_t link_nsid = ns_id;
 	uint8_t bypass = 0;
 
+	frrtrace(2, frr_zebra, netlink_interface, h, ns_id, startup);
```

Comment:
you have 3 parameters, so you need to use 3

Reply:
fixed